src/components/beat.js

```diff
@@ -537,7 +537,8 @@ AFRAME.registerComponent('beat', {
 					if (
 						bombs[i].spawnTime < data.time + 0.01 &&
 						bombs[i].spawnTime > data.time - 0.01 &&
-						(!bombs[i].id || bombs[i].id == data.noteId || bombs[i].id == data.noteIdWithScoring)
+						// We only care about matching lineIndex*1000 and noteLineLayer*100 from the ID
+						(!bombs[i].id || Math.floor((bombs[i].id % 10000) / 100) == Math.floor((data.noteId % 10000) / 100))
```

Comment:
I hope this makes sense; it works but I'm not sure why the IDs don't fully match. In my testing, this has been broken since 8f66bb7364520757d1ff89bed559434258af470e. Before the ID check it was incorrectly highlighting all the bombs with the same spawn time.